composer.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "run_as_root/ext-magento2-google-shopping-feed",
-    "version": "1.0.0",
```

Comment:
we should extract that into a separate PR if this PR takes to long to merge as it fixes a different issue, isn't it?

Reply:
Done.